src/java/Transport.java

```diff
@@ -0,0 +1,10 @@
+package java;
+
+public abstract class Transport {
+    public int power;
+    public int weight;
+    public int maxSpeed;
+    public String mark;
+
+}
+
```

Comment:
@Ladauser Лишние пустые строки
```suggestion
}
```